pkgpanda/docs/activating.md

```diff
@@ -1,11 +1,13 @@
-# Activating Packages `pkgpanda activate`
+# Activating Packages
 
-The list if currently active packages is stored in 'active.json' inside of INSTALL_ROOT.
+`pkgpanda activate <id>... [options]` will activate the list of packages give as `id`s in the command line.
```

Comment:
I think something like `... will activate the list of package id's.` reads better.